cft_sidecar_failopen.yaml

```
@@ -52,6 +58,10 @@ Parameters:
     Type: String
     Description: "Port allocated on the sidecar for this repository."
 
+  RepoType:
+    Type: String
+    Description: "The type of the repo that will be tested"
+    AllowedValues: [mysql, pg]
```

```suggestion
    AllowedValues: [mysql, postgresql]
```

Using same conventions as defined here: https://github.com/cyralinc/terraform-cyral-sidecar-aws/blob/main/variables_sidecar.tf#L141